libgammu/phone/at/atgen.c

```diff
@@ -2905,6 +2905,17 @@ GSM_Error ATGEN_ReplyGetPacketNetworkLAC_CID(GSM_Protocol_Message *msg, GSM_Stat
 			&rac, sizeof(rac) /* Routing Area Code, ignored for now */
 			);
 
+	/* Reply without ACT */
+	error = ATGEN_ParseReply(s,
+			GetLineString(msg->Buffer, &Priv->Lines, 2),
+			"+CGREG: @i, @i, @r, @r, @i",
+			&i, /* Mode, ignored for now */
+			&state,
+			NetworkInfo->PacketLAC, sizeof(NetworkInfo->PacketLAC),
+			NetworkInfo->PacketCID, sizeof(NetworkInfo->PacketCID),
+			&act, /* Access Technology, ignored for now */
```

Comment:
The comma here is causing syntax error on some compilers (see Windows build failure on AppVeyor):

> libgammu\phone\at\atgen.c(2917): error C2059: syntax error: ')'